changelog.d/20241017_134221_dmitrii.lavrukhin_fix_invalid_tracks.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Track shape duplication on import
```

Comment:
Now there should be no errors during export tracks with duplicated shapes. Why not mention that?